pom.xml

```diff
@@ -148,6 +148,16 @@
             <scope>test</scope>
             <version>20180130</version>
         </dependency>
+        <dependency>
+            <groupId>fish.payara.extras</groupId>
+            <artifactId>payara-embedded-all</artifactId>
+            <version>5.2020.4</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.jersey</groupId>
+            <artifactId>jersey-client</artifactId>
+            <version>1.18</version>
```

Comment:
is this library used only for multipart mime? can we have our implementation that does that part only?
the reason being is we are adding library that is only used in 1 use case, and it will make the SDK heavy for the integrators cause the will be added on top of their current library deps